build/media-suite_compile.sh

```diff
@@ -1688,7 +1688,7 @@ if [[ $ffmpeg != "no" ]]; then
 
         if [[ ${#FFMPEG_OPTS[@]} -gt 25 ]]; then
             # remove redundant -L and -l flags from extralibs
-            do_patch "ffmpeg-0001-configure-fix-failures-with-long-command-lines.patch"
+            do_patch "https://jb-alvarado.github.io/media-autobuild_suite/ffmpeg-0001-configure-fix-failures-with-long-command-lines.patch"
```

Comment:
Is this the right URL?

Reply:
```suggestion
            do_patch "https://jb-alvarado.github.io/media-autobuild_suite/patches/ffmpeg-0001-configure-fix-failures-with-long-command-lines.patch"
```

Reply:
I miscopied the link, I hate fat fingers with phone screens